src/util/BitField.js

```diff
@@ -130,11 +130,10 @@ class BitField {
 
   /**
    * Data that can be resolved to give a bitfield. This can be:
-   * * A string (see {@link BitField.FLAGS})
-   * * A bit number
+   * * A bit number (this can be a number literal or a value taken from {@link BitField.FLAGS})
    * * An instance of BitField
    * * An Array of BitFieldResolvable
-   * @typedef {string|number|BitField|BitFieldResolvable[]} BitFieldResolvable
+   * @typedef {number|BitField|BitFieldResolvable[]} BitFieldResolvable
```

Comment:
if its still accepted why remove it from the docs?

Reply:
So people use the FLAGS object more instead of strings that you can easily mistype

Reply:
We should be fine to treat this as a hidden "feature".

Typescript will still tell you that you can use a string and the method itself won't error if you do.

Reply:
fair enough